### ISSUE TITLE
+ and - sign not visible in latest version

### ISSUE THREAD

**Issue by adityamalla** [fooplugins/FooTable#185]
Hi,

Instead of the + and - signs for opening and closing the hidden tables, I see Chinese signs.

This is in the demo's. But unfortunately also in my design.

How do I get the + and - back for the mobile view for opening and closing the hidden tables!?

Thanks in advance

**Comment by bradvin** [comment]
Hi @adityamalla - what browser are you seeing this behavior?

**Comment by adityamalla** [comment]
Firfox 26.0

**Comment by kedano** [comment]
Hi, just tested a site that use Footable in Firefox 26, and the whole table is dysfunctional, not just the icons. I'll investigate and report back.

**Comment by adityamalla** [comment]
Thank you Kedano...

**Comment by kedano** [comment]
On our project the issue was that I've positioned the icons absolute within the table cell to have full control over the icon position. 

Apparently position:relative is not valid CSS on a `<td>`.

**Comment by adityamalla** [comment]
Guys, 
I got the solution for the above mentioned issue. Actually a default content value is set in the footable.core.css as shown below,

.footable.breakpoint > tbody > tr > td > span.footable-toggle:before {
  content: "\000";
}

And I have changed the  conetnt value and added one more for "-" symbol in the footable.core.css as shown below,

.footable.breakpoint > tbody > tr > td > span.footable-toggle:before {
  content: "\002b";
}
.footable.breakpoint> tbody > tr.footable-detail-show > td > span.footable-toggle:before {
  content: "\2212";
}

And i got the plus and minus signs in my grid successfully. And i got the content values from bootstrap-glyphicons.css.

Thanks kedano and Bradvin for you approach and help. :)
